src/components/App/App.js

```diff
@@ -16,6 +16,7 @@ import SingleFull from '../../views/Single/SingleFull';
 import SinglePage from '../../views/Single/SinglePage';
 import Doc from '../../views/Doc/Doc';
 import Colors from '../../views/Colors/Colors';
+import Icons from '../../views/Icons/Icons';
```

Comment:
`Icons` will be a bit confusing with `components/Icon/Icon` and `components/Icon/Icons`...